### PR TITLE
Fix assorted UG flaws

### LINKED ISSUE #194: [PE-D][Tester E] Misspelling in the UG
This is just a typo to take note!

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: freshcabbage123/ped#3

### LINKED ISSUE #216: [PE-D][Tester A] Filter by course [Taken] bug
"[taken] (optional). Only for filtering by course, this parameter additionally filters out contacts who have finished the course."
In the documentation, it isn't really clear how a course is defined to be finished, so I assumed that is based by the graduation date, but the image above shows that Charlotte has already graduated in 2000, which means she should have finished all the course, but the filter list still shows her name .

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: Blizzeracz/ped#5

### PR DESCRIPTION
**Pull request by Eola-Z**
Fixes #194 
Fixes #216

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -394,12 +394,15 @@ Parameters:
 in the filtered contact list must have part of the `[term]` in the `[field]`
 specified above.
 * `[taken]` (optional). Only for filtering by `course`, this parameter additionally filters
-  out contacts who have finished the course.
+  out contacts who have finished/haven't started the course at the current date of your system.
```
Comment (awhb) [inline docs/UserGuide.md]: "as of" seems like the better phrase. Are you ok to change this?

**Review by awhb — CHANGES_REQUESTED** [review]
Just a minor nit!

File: docs/UserGuide.md (hunk 1 above)
Comment (Eola-Z) [reply docs/UserGuide.md]: Noted, and fixed 👍

File: docs/UserGuide.md [hunk 2]
```diff
@@ -394,12 +394,16 @@ Parameters:
...
 * `[taken]` (optional). Only for filtering by `course`, this parameter additionally filters
-  out contacts who have finished the course.
+  out contacts who have finished/haven't started the course as of the current date of your system.
 
     List of options:
 
-    * `true` - Remove contacts that have finished taking the course.
-    * `false` - (default). Include contacts who have finished taking the course.
+    * `true` - Remove contacts whose courses' start and end date range are outside 
+    your system's current date.
```
Comment (nknguyenhc) [inline docs/UserGuide.md]: Shouldn't the phrasing for this part be the same as above (line 397), and also this phrasing may sound a bit technical.

**Review by nknguyenhc — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -421,12 +421,15 @@ Parameters:
 in the filtered contact list must have part of the `[term]` in the `[field]`
 specified above.
 * `[taken]` (optional). Only for filtering by `course`, this parameter additionally filters
-  out contacts who have finished the course.
+  out contacts who have finished/haven't started the course as of the current date of your system.
```
Comment (awhb) [inline docs/UserGuide.md:424]: Not sure why but this seems very confusing to me. `taken` refers to those who are currently taking the course?

**Review by awhb — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 3 above)
Comment (Eola-Z) [reply docs/UserGuide.md:424]: Yeah, I've thought about changing the term to 'taking', but does that count as a feature change?

Comment (nknguyenhc) [reply docs/UserGuide.md:424]: We have to keep it as it is, the best thing we can do now is to explain it clearly in the UG.

Comment (awhb) [reply docs/UserGuide.md:424]: ok

**Review by awhb — APPROVED** [review]
LGTM!

**Pull request merged**